sdk/appplatform/azure-mgmt-appplatform/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Release History
 
+## 9.1.1 (2024-10-25)
+
+### Other Changes
+
+  - This package is converted to single-api package which only contain one kind of api version. After this change, the package size will be much smaller then before. Since most of users actually only use the API of default api-version, you won't even feel the change.
+
```

Comment:
another version: `This package has removed all other api versions except the default one, and would only contain one kind of api version
henceforth. For users who only use the API of default api-version, this change is ignorable.`

Reply:
it's still important for us to stick to semantic versioning and do a major bump, but I think we can definitely say "if you're using the default sdk, you won't be broken"